From stable-bounces@linux.kernel.org Wed Nov 14 16:59:13 2007
From: Li Zefan <lizf@cn.fujitsu.com>
Date: Wed, 14 Nov 2007 16:58:33 -0800
Subject: drivers/video/ps3fb: fix memset size error
To: torvalds@linux-foundation.org
Cc: Geert.Uytterhoeven@sonycom.com, akpm@linux-foundation.org, lizf@cn.fujitsu.com, stable@kernel.org
Message-ID: <200711150058.lAF0wXPr002540@imap1.linux-foundation.org>


From: Li Zefan <lizf@cn.fujitsu.com>

patch 3cc2c17700c98b0af778566b0af6292b23b01430 in mainline.

The size passing to memset is wrong.

Signed-off-by Li Zefan <lizf@cn.fujitsu.com>
Acked-by: Geert Uytterhoeven <Geert.Uytterhoeven@sonycom.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/video/ps3fb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/video/ps3fb.c
+++ b/drivers/video/ps3fb.c
@@ -659,7 +659,7 @@ static int ps3fb_blank(int blank, struct
 
 static int ps3fb_get_vblank(struct fb_vblank *vblank)
 {
-	memset(vblank, 0, sizeof(&vblank));
+	memset(vblank, 0, sizeof(*vblank));
 	vblank->flags = FB_VBLANK_HAVE_VSYNC;
 	return 0;
 }
